.python-version

```diff
@@ -1 +1 @@
-3.6.5
+3.7.5
```

Comment:
Maybe you've [already seen](https://circleci.com/gh/equinor/gordo-components/14228?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link), but CircleCI's pyenv only have `3.7.0` and I'm not sure if it's better to see how to change that fact or be 'good-enough' with `3.7`

Reply:
Yeah, I vaguely recall something about this. Thanks! :eyes: